src/main/java/racingcar/InputView.java

```diff
@@ -1,24 +1,51 @@
 package racingcar;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Scanner;
 
 public class InputView {
 
+    private static final String SPLIT_REGEX = ",";
+
+    private final Scanner scanner = new Scanner(System.in);
+
     /**
      * 차의 대수를 입력받습니다.
      */
-    public RacingCarDriver inputCarAmount(Scanner scanner) {
+    public RacingCarDriver inputCarAmount() {
         System.out.println("자동차 대수는 몇 대 인가요?");
-        return new RacingCarDriver(scanner.nextInt());
+        return new RacingCarDriver(this.scanner.nextInt());
     }
 
     /**
      * 몇 회차까지 진행 할 지 입력받습니다.
      * @return round
      */
-    public int inputRacingRound(Scanner scanner) {
+    public int inputRacingRound() {
         System.out.println("시도할 회수는 몇 회 인가요?");
-        return scanner.nextInt();
+        return this.scanner.nextInt();
+    }
+
+    /**
+     * 참가자 리스트를 받습니다.
+     * @return round
+     */
+    public NamedRacingCarDriver inputParticipants() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+        return new NamedRacingCarDriver(this.makeParticipantsList(this.scanner.nextLine()));
+    }
+
+    /**
+     * @param inputs
+     * @return
+     */
+    public String[] makeParticipantsList(String inputs) {
```

Comment:
```suggestion
    public List<String> makeParticipantsList(String inputs) {
        return Stream.of(inputs.split(InputView.SPLIT_REGEX))
                .map(String::trim)
                .collect(Collectors.toList());
    }
```

Stream을 활용하시면 코드를 좀더 깔끔하게 작성할 수 있습니다

Reply:
Stream을 활용하도록 변경하였습니다~